json_merger/comparator.py

```diff
@@ -29,8 +29,17 @@
 
 
 class BaseComparator(object):
+    """Abstract base class for Entity Comparison."""
 
     def __init__(self, l1, l2):
+        """
+        Do not override the constructor since it will be called by the
+        json_merger logic.
```

Comment:
Let me do my stupid comment any way. :-P I can't really understand the whole rest of the refactoring unless I can focus on it for 24 hours in a row :-P